content/kubermatic/main/tutorials-howtos/audit-logging/_index.en.md

```diff
@@ -106,12 +142,10 @@ Depending on your architecture, it might be advisable to use the sidecar configu
 filters:
   - Name: record_modifier
     Match: *
-    Record: cluster <CLUSTER ID>
+    Record: usercluster ${CLUSTER_NAME}
 ```
 
-Replace `<CLUSTER ID>` with the ID of your cluster.
-
-Future KKP releases may add an environment variable to automatically get the cluster ID or even enrich records with this information by default.
+The `${CLUSTER_NAME}` environment variable is automatically set to the ID of your cluster by KKP.
```

Comment:
nit: I think we should keep the CLUSTER_ID name since it's actually the cluster ID.

Reply:
yeah, good idea! @buraksekili can you please change in KKP code from CLUSTER_NAME to CLUSTER_ID (which it actually is).

Reply:
Thanks @moelsayed and @kron4eg! Raised a PR on KKP to rename the environment variable: https://github.com/kubermatic/kubermatic/pull/14459

Reply:
updated the docs accordingly in 27de2fcb071e6e2c0e96a83a1b9e07448ee38036